libr/core/agraph.c

```diff
@@ -2310,12 +2310,14 @@ static void add_child(RCore *core, RAGraph *g, RANode *u, ut64 jump) {
 	}
 	char *title = get_title (jump);
 	RANode *v = r_agraph_get_node (g, title);
-	ut64 a = r_num_get (NULL, u->title);
-	ut64 b = r_num_get (NULL, title);
+	if (v) {
+		ut64 a = r_num_get (NULL, u->title);
+		ut64 b = r_num_get (NULL, title);
+		const char *k = sdb_fmt ("agraph.edge.0x%" PFMT64x "_0x%" PFMT64x ".highlight", a, b);
```

Comment:
better not to use sdb_fmt as its not thread safe, we are in the plan to remove its uses and at some point have a simpler string-in-stack api like we did 2 years ago without reaching any agreement. but it will definitvely make the code cleaner.

Reply:
What should I use instead? Just `r_str_newf` a string that would of been returned by sdb_fmt? 

I am not too familiar with the inner workings of sdb.

Reply:
Use snprintf for now. Strnew uses the heap and there was 3 r_strf proposals that all of them ens up deleted. But will be good to bring one of them